pkg/controller/README.md

```diff
@@ -0,0 +1,17 @@
+### Controller local test
+#### Run inside the cluster
+1.run make docker.all to push image with your HUB and TAG
```

Comment:
can we add a step on specify HUB and TAG first?

Reply:
also whether user needs to export BUILD_WITH_CONTAINER

Reply:
ok, done

Reply:
BUILD_WITH_CONTAINER is default.  I am not sure HUB nor TAG are passed into the make core file.  I will investigate this and fix it if it is busted.

Reply:
@sdake yes actually I did not use BUILD_WITH_CONTAINER for this step, but looks like you already added passing HUB and TAG in common files. 

Also another reason I failed to run make docker.all with container was some docker permission issue, but other make target works fine.